x/staking/src/keys.rs

```diff
@@ -0,0 +1,19 @@
+use gears::types::address::BaseAddress;
+
+pub fn length_prefixed_addr_key<const PREFIX: u8>(addr: &BaseAddress<PREFIX>) -> Vec<u8> {
+    let addr_str = addr.to_string();
+    let bytes = addr_str.as_bytes();
+    let mut bytes_prefix = bytes.len().to_ne_bytes().to_vec();
```

Comment:
`to_ne_bytes` isn't deterministic across nodes - a different result may be returned on different machines. Is this what the Cosmos SDK does?

Reply:
It doesn't. I'll change to `to_le_bytes`.